config/jobs/driverkit/driverkit-test.yaml

```diff
@@ -0,0 +1,21 @@
+presubmits:
```

Comment:
We should double check if we want this job to be a local job or not. All `.yaml` files will get merged into real prowjobs.

Reply:
Completely agree.
For conciseness I removed this driverkit-itself build jobs (not `build-drivers`), as it's out of scope of this PR IMHO.